tests/client/test_auth.py

```diff
@@ -523,17 +523,6 @@ async def test_digest_auth_qop_including_spaces_and_auth_returns_auth(qop: str)
     assert len(response.history) == 1
 
 
-@pytest.mark.anyio
-async def test_digest_auth_qop_auth_int_not_implemented() -> None:
-    url = "https://example.org/"
-    auth = httpx.DigestAuth(username="user", password="password123")
-    app = DigestApp(qop="auth-int")
-
-    async with httpx.AsyncClient(transport=httpx.MockTransport(app)) as client:
-        with pytest.raises(NotImplementedError):
-            await client.get(url, auth=auth)
-
-
```

Comment:
Instead of removing this test case, let's change it so that we're testing `auth-int`.